### PR TITLE
[UPDATE] Keep selling for b2b

### PR DESCRIPTION
**Pull request by nishantdania**
On hold until the feature is released to all the customers

### CODE REVIEW

File: locales/variant.en.yml [hunk 1]
```diff
@@ -291,7 +293,14 @@ en:
       keep_selling: {
         name: "keep_selling",
         type: "Boolean",
-        description: "Allow item to continue to be sold, even if it is no longer in stock",
+        description: "Allow item to continue to be sold on Integrated sales channels, even if it is no longer in stock",
+        updatable: true,
+        creatable: true,
+      },
+      keep_selling_b2b: {
```
Comment (ryancyq) [inline locales/variant.en.yml:300]: i am not sure whether this is the API we want to release to the API user.

Comment (nishantdania) [reply locales/variant.en.yml:300]: @MohammadHabbab Should we expose this API or keep it internal only ?

Comment (nishantdania) [reply locales/variant.en.yml:300]: I would lean towards exposing this since there would be no other way for API users to change this setting otherwise

Comment (bradleypriest) [reply locales/variant.en.yml:300]: I'd tend to keep it private in the short term, until we work out the timelines for building out the wider per-channel attribute functionality and what that API looks like.

Comment (nishantdania) [reply locales/variant.en.yml:300]: Ok, closing this PR for now in that case

**Pull request closed without merging**